### DISCUSSION TITLE
文管地址栏的file://能不能去掉

### DISCUSSION THREAD

**Discussion by a137933** [linuxdeepin/developer-center#2925] [Features Request & Ideas | 特性请求 & 头脑风暴]
比如我打开opt目录，此时复制地址，复制出来的是“file:///opt”，这个路径在终端是识别不了的，能不能去掉file://这个前缀？

**Comment by deepin-CD** [comment]
在文管的地址栏中，按CTRL + L，即可复制没有前缀的地址。

**Comment by BLumia** [comment]
<kbd>Ctrl + L</kbd> 即可编辑地址栏，这时候的地址就会是没有 file:// 协议前缀的路径了。
